modules/azure/api_connectors/key_vault_managed_identity/connection.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.9.1.41621",
+      "templateHash": "14233748139416635108"
+    }
+  },
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "location"
+      }
+    },
+    "keyvault_connection_name": {
+      "type": "string",
+      "metadata": {
+        "description": "Name to use for this connection"
+      }
+    },
+    "key_vault_name": {
+      "type": "string",
+      "metadata": {
+        "description": "Service Bus namespace"
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Web/connections",
+      "apiVersion": "2016-06-01",
+      "name": "[parameters('keyvault_connection_name')]",
+      "location": "[parameters('location')]",
+      "kind": "V1",
+      "properties": {
+          "displayName": "[parameters('keyvault_connection_name')]",
+          "parameterValueType": "Alternative",
+          "alternativeParameterValues": {
```

Comment:
Do you need this too ?

"parameterValueSet": {
          "name": "managedIdentityAuth",

Reply:
Ok :)